packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/angular/basic-sign-in.md

```diff
@@ -0,0 +1,86 @@
+This section helps you review the simple password-only sign-in use case from the sample app. This use case is outlined in the following sequence diagram with your single-page app as the client:
+
+![Sequence diagram that displays the back and forth between the resource owner, sdk, authorization server, and resource server for a basic SPA password sign-in flow.](/img/oie-embedded-sdk/password-only-spa-authjs-flow.svg "Auth JS + SPA password-only sign-in flow")
+
+### Review the Okta configuration settings
+
+Review the `src/app/okta-config.ts` file that references the required [configuration settings](#configuration-settings) to initialize your Okta Auth JS instance. The `okta-config.ts` file references the values from your Okta app integration.
+
+```TypeScript
+import { OktaAuth } from '@okta/okta-auth-js';
+import { FactoryProvider, InjectionToken } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import { Router } from '@angular/router';
+
+const oidcConfig = {
+  clientId: '${clientId}',
+  issuer: 'https://${yourOktaDomain}/oauth2/default',
+  scopes: ['openid', 'profile', 'email'],
+  pkce: true
+};
+```
+
+### Instantiate the Okta Auth JS client
+
+Review the same Angular `src/app/okta-config.ts` file that imports required libraries and instantiates the Okta Auth JS client with values from the `oidcConfig` variable.
+
+```TypeScript
```

Comment:
The simplified code for this could be

```ts
const oktaAuthFactory = () => {
  return new OktaAuth({...oidcConfig});
}
```

If we want to show the entire code in `okta-config.ts` that also makes sense to me, but it does have specialized handling for already logged in and magic link scenarios.

Probably should also add the code to add the OktaAuth to the Angular app in `app.module.ts`:

```ts
import { OKTA_PROVIDER } from './okta-config';

@NgModule({
  providers: [
    OKTA_PROVIDER
  ]
})
```

If we want to simplify, we can also not create the factory provider, instantiate the OktaAuth and provide to the app directly in the app.module.ts.

Reply:
As you mentioned I added the full code from `okta-config.ts`